website/markdown/docs/contribution/performance.mdx

```diff
@@ -11,7 +10,7 @@ To test out generation speeds and to provide some utilities to aid profiling Tui
 - `fixturegen`: A tool to generate large fixtures
 - `tuistbench`: A tool to benchmark Tuist
 
-Those tools are located within the [`tools/`](https://github.com/tuist/tuist/blob/master/tools) directory. 
+Those tools are located within the [`tools/`](https://github.com/tuist/tuist/blob/master/tools) directory.
```

Comment:
do you have a code formatter for markdown - seems to have trimmed a lot of whitespace from various non source code files :/

Reply:
I have [prettier](https://prettier.io/) set-up with